2021-offseason/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -42,7 +46,19 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
+
+  //Assign operator controller buttons A and Y to startIntake and startReverse respectively
   private void configureButtonBindings() {
+    // A button -> run intake forward
+    new JoystickButton(operatorController, XboxController.Button.kA.value)
+    .whenPressed(new RunCommand(s_intake::startIntake, s_intake))
+    .whenPressed(new RunCommand(s_intake::stopIntake, s_intake));   
+
+    // B button -> run intake reverse
+    new JoystickButton(operatorController, XboxController.Button.kB.value)
+    .whenPressed(new RunCommand(s_intake::startReverse, s_intake))
+    .whenPressed(new RunCommand(s_intake::stopIntake, s_intake)); 
```

Comment:
nit: like the hopper code, indent these 2 lines forward more
same with A button